eng/tools/ChangeValidation/ChangeValidation.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
```

This should probably be `$NetCurrent`

```suggestion
        <TargetFramework>$(NetCurrent)</TargetFramework>
```